src/components/screenings/ScreeningCard.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+    <div class="screening" v-if="!!movieSeances.length">
+        <img class="screening__image" :src="movie.poster_url" />
+        <div class="screening__info">
+            <h1 class="screening__info--title">{{ movie.title }}</h1>
+            <div class="screening__info__details">
+                <base-chip class="screening__info__details--genre">
+                    {{ movie.genre.name }}
+                </base-chip>
+                <p class="screening__info__details--length">{{ movieLength }}</p>
+            </div>
+        </div>
+        <div class="screening__seances">
+            <div class="screening__seances--seance" v-for="seance in movieSeances" :key="seance.id">
+                {{ getSeanceStartHour(seance.datetime) }}
+            </div>
+        </div>
+     </div>
+</template>
+
+<script>
+import api from "@/api/FactoryRepository"
+import {getFormattedDate, formatMovieLength} from '@/helpers/dateHelper'
+const seances = api.get('seances')
+export default {
+    props: {
+        movie: {
+            type: Object
+        },
+        filterBy: {
+            type: String
+        }
+    },
+    data() {
+        return {
+            movieSeances: [],
+        }
+    },
+    methods: {
+        async getFilteredSeances() {
+            const { data } = await seances.getSeancesByMovieAndDate({movieId: this.movie.id, date: this.filterBy || getFormattedDate(new Date())});
+            this.movieSeances = data
+        },
+        getSeanceStartHour(seanceDate) {
+            const date = new Date(seanceDate)
+            const hour = ('0' + date.getHours()).slice(-2)
+            const minutes = ( '0' + date.getMinutes()).slice(-2)
+            return hour + ":" + minutes
+        }
+    },
+    computed: {
+        movieLength() {
+            return formatMovieLength(this.movie.length)
+        },
+    },
+    async created() {
+        await this.getFilteredSeances()
+    },
+    watch: {
+        filterBy() {
+            this.getFilteredSeances()
+        }
```

Comment:
[just a tip you can ignore] instead of `created` and watcher you can have just a watcher with `immediate: true`

> Passing in `immediate: true` in the option will trigger the callback immediately with the current value of the expression (https://v2.vuejs.org/v2/api/?redirect=true#vm-watch)

so effectively: when created

you can st it like this: 
```js

filterBy: {
  immediate: true,
  handler (newValue, oldValue) {
    this.getFilteredSeances()
  }
}
```